### ISSUE TITLE
Training | Purchase Card Training for A/OPCs

### ISSUE THREAD

**Issue by JessicaMarine1** [GSA/smartpay-training#59]
As a user of the site, I want to access Purchase Card Training for A/OPCs without registering (providing an email), so I can view/print training materials.

**Pre-Condition:**
- Training material content is complete
- Content available [HERE](https://docs.google.com/document/d/1YUB-2HEHp53Am2e1r8qvOJtb9I89h5tu2pdIB5J3R7o/edit)

**Acceptance criteria:**
- [ ] Users can progress through each lesson of the training in the browser without providing an email address
   - Note: this should have the same exact look / experience as the [Travel Card Training for Card/Account Holders & Certifying Officials](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/site/gsa/smartpay-training/training_travel/)
- [ ] Users can access quiz registration form from end of training material
- [ ] Users can view & print a 508 compliant, print-friendly version of the full training

**Comment by A-L33** [comment]
@brentryanjohnson Content ready for review.

**Comment by brentryanjohnson** [comment]
@A-L33 ✅ Reviewed!

**Comment by A-L33** [comment]
@weiwang-gsa Ready!

**Comment by weiwang-gsa** [comment]
@A-L33 I have following questions:

1. Lesson 1: 

- "What is a GSA SmartPay Purchase card/account?" is before "What are some benefits of using the GSA SmartPay Purchase card/account?" in AOPC lesson one, but in card holder purchase training lesson 1, it is after the benefits and stay with all what is xxx accounts. ( https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/site/gsa/smartpay-training/training_purchase/lesson01/), just want to confirm if this is correct or need to be consistent.
- "What is a Centrally Billed Account (CBA) and an Individually Billed Account (IBA), and why is it important to know the difference?" there is comma , before and, but in card holders, the comma is missing. Do I need to add comma to the card holder purchase training materials too?

AOPC purchase training title below
![Image](https://github.com/GSA/smartpay-training/assets/119954777/1728a71a-4e50-434f-bfa3-dac84ec1618d)

card holder purchase training title below:
![Image](https://github.com/GSA/smartpay-training/assets/119954777/db9ab66b-b675-444e-a028-9e7caf0587c2)


 
2. I found there are links to the old smartpay website URLs, which needs to be updated once the website development is done, maybe we should have separate user story to track the link update once website development is done? For this ticket, I just use whatever doc provided, but it needs code update afterwards.

**Comment by A-L33** [comment]
@weiwang-gsa Just touched base with Rebekah, she confirmed the content order.  Note that we may be doing some content cleanup for the other loaded trainings later, so there might be some inconsistencies that we'll fix later.  Thank you!

**Comment by brentryanjohnson** [comment]
@weiwang-gsa You can delete the comma in the version that has it. Grammatically, it should be there, because it separates two independent clauses, but it also impedes readability in this context, so feel free to remove it.

**Comment by weiwang-gsa** [comment]
@melchoyce This issue doesn't involve any design change, use the same design as other trainings, anyway it is ready for design review: https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/site/gsa/smartpay-training/training_purchase_pc/

**Comment by JessicaMarine1** [comment]
@weiwang-gsa one thing I see is the "(508 compliant)" parenthetical is still present for the download full training feature. The decision was made to remove that. If this is a reusable component, it's OK to make the change for all. If not, we have it noted in another story to go back and change it on the previously implemented trainings, as needed.

**Comment by weiwang-gsa** [comment]
@JessicaMarine1 Thanks for catching it, I made code change to remove it from all existing trainings, once Jeff approve the PR, I will merge it into main branch.

**Comment by melchoyce** [comment]
Looks good to me from a design perspective!

**Comment by melchoyce** [comment]
Actually—just noticed this is purchase card training, so the header and selected sidebar item should be using the Purchase Card red color, `#900028`:

![image](https://github.com/GSA/smartpay-training/assets/2846578/b8b517cb-6a7a-41af-b743-192771b02083)

Same goes for the quiz header in #77.

**Comment by weiwang-gsa** [comment]
@JessicaMarine1 @melchoyce For header banner dynamically change the background color, I think it needs separate user story, currently the banner background color is fixed and all training using same banner. And I assume the background color need to be changed for fleet training as well.

**Comment by melchoyce** [comment]
Yeah, good idea. Can we get a new story to tackle that in a future sprint @JessicaMarine1?

**Comment by JessicaMarine1** [comment]
@melchoyce could we add that to the global design changes needed during code freeze?

**Comment by melchoyce** [comment]
Yeah, that should work great 👍

**Comment by JessicaMarine1** [comment]
@A-L33 this is ready for your review.

**Comment by A-L33** [comment]
Done!
